frontend/package.json

```diff
@@ -10,21 +10,14 @@
     "file-saver": "^1.3.8",
     "rc-slider": "^8.6.1",
     "react": "^16.4.1",
-    "react-dom": "^16.4.1",
+    "react-dom": "^16.4.2",
```

Comment:
should this also be changed in the devDependencies section below?

Reply:
never mind , it is a different package "@types/react-dom"

Reply:
Its not in the devDependencies section, except for `@types/react-dom`, is this what you mean?